docs/modules/ROOT/pages/high-availability-mode.adoc

```diff
@@ -0,0 +1,59 @@
+= Running Highly Availiable Hazelcast clusters
+:description: High Availability Mode guarantees that even if the Kubernetes node or whole availability zone is down (and all related Hazelcast members are terminated), you will not experience any data loss.
+
+Hazelcast offers the Partition Group Configuration and its types are ZONE_AWARE and NODE_AWARE, which forces Hazelcast members to store the given data partition backup inside a member located in a different availability zone or Kubernetes node. To learn about more, see the xref:hazelcast:clusters:partition-group-configuration.adoc[Platform documentation].
+
+When using the ZONE_AWARE and NODE_AWARE partition grouping, a Hazelcast cluster spanning multiple availability zones and nodes should have an equal number of members in each availability zones and node. Otherwise, it results in uneven partition distribution among the members.
+
+At the Platform Operator, users can configure type of partition-group and required Kubernetes scheduling policy(`topologySpreadConstraints`) to distribute members across availability zones and nodes automatically via modifying single `highAvailabilityMode` parameter.
+
+{description}
+
+== Configuring High Availability Mode 
+
+Below are the configuration options for the High Availability Mode feature.
```

Comment:
In the other topics, we seem to have a standard line for the introduction of a new configuration item. Could we add something like this, if relevant? You can use the following configuration options in your XXXXX resource. For more detailed information, see the [API Reference](XXXXXXX).

Reply:
This feature has only one parameter so I think we do not need to redirect user API reference. WDYT?

Reply:
Yes, that makes sense.